poll/poll.py

```diff
@@ -876,6 +883,13 @@ def studio_submit(self, data, suffix=''):
 
         return result
 
+    @XBlock.json_handler
+    def student_voted(self, data, suffix=''):
+        return {
+            'voted': self.get_choices() is not None,
+            'private_results': self.private_results
+        }
+
```

Comment:
Given that the two implementations are identical, can't this be added to `PollBase`?